### ISSUE TITLE
Media Info Source File is added twice to log

### ISSUE THREAD

**Issue by Zodler** [staxrip/staxrip#1221]
In one of the previous versions, Media Info Source File was not added to the log anymore.

Now it is added twice. Here are the 2 places that it is added

log file:

System Environment
Media Info Source File <----------------
Demux MKV

Configuration
Media Info Source File <----------------
AviSynth Script

**Comment by Dendraspis** [comment]
This is intended as the first one is read out when you open the source file and the second time is when you start the processing.

**Comment by Zodler** [comment]
But why? Why it is intended? There is no need to have the same exact information twice for no reason. In the previous versions it was not like this. As I said they are character by character the same exact information that take extra space.

**Comment by Zodler** [comment]
I'm interested to understand but unfortunately I still don't understand what you said. Could you please explain it again in another way?

What scenario? Please give an example. How could the source be different in the second segment?

**Comment by Dendraspis** [comment]
For example when it gets overwritten by another app, mostly the one that created the source file, especially when you created it automatically or by script.

Just one possible scenario:
You have 20 jobs running, takes you about 21 days. After one week you decide you want to change an audio track of `XYZ.mkv`. You do so and then you open `XYZ.mkv` again in StaxRip to add it to the job list without removing the old job. After 5 days you get an error message and you decide to open a bug report on GitHub.

That it can't happen in your workflow doesn't mean that it can't happen anybody else. And yes, it happened.

Basically a `Media Info` right before the processing would be enough, but it is also very useful when it comes to debugging error messages before the processing can start.
